codecov.yaml

```diff
@@ -0,0 +1,22 @@
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+    patch:
+      default:
+        target: auto
```

Comment:
Are these default settings? If so, will we remove them?

Reply:
https://docs.codecov.com/docs/codecov-yaml#default-yaml

From the documentation, I think it's not default settings.

Reply:
Yes. These settings can be changed at any time if it is no longer useful.